README.md

```diff
@@ -22,6 +22,14 @@ go build github.com/giantswarm/awstpr
 However, since this project is just a specification used by the other projects, there only goal of building is to check
 whether the build is successful. This is just a library which needs to be vendored by the project aiming to use it.
 
+### Run the tests
+
+To run the tests.
+
+```
+go test $(glide novendor)
```

Comment:
With golang 1.9.0, we don't need glide anymore - `./...` doesn't match vendor.

So, this can totally just read `go test ./...`

Reply:
Done. Good catch that's really nice! <3

Reply:
My favourite feature in 1.9 :D

Reply:
Nice! :100: